update_compilers/install_libraries.sh

```diff
@@ -240,4 +240,65 @@ install_lua() {
     done
 }
 
-install_lua v5.3.5 v5.4.0
+#install_lua v5.3.5 v5.4.0
+
+install_nsimd() {
```

Comment:
We're trying to avoid adding new bash script code here. I'll defer to @partouf but hopefully there's a YAML-based equivalent or build we can add to make this easier to test and update.

Reply:
I recommended this because it's a shared C library with multiple architectures and there isn't really a way of doing this through the yaml or the builder.

Reply:
Perfect! We can merge it once the commented-out stuff is there -- we can revisit this as we port more things to the YAML way (And this is 90% there already).